### PR TITLE
Louca/misc

### PR DESCRIPTION
**Pull request by loucadufault**

### CODE REVIEW

File: kanbananza-backend/src/main/controllers/column.js [hunk 1]
```diff
@@ -18,17 +16,38 @@ const create = async (req, res, next) => {
...
+//         req.query.name
+//       );
+//     } else {
+//       cards = await cardService.findAllColumnCards(req.params.id);
+//     }
+//   } catch (e) {
+//     next(e);
+//   }
+//   res.status(200).json(cards.map((card) => card.toDTO()));
+// };
```
Comment (loucadufault) [inline kanbananza-backend/src/main/controllers/column.js:45]: pending card lower layers

File: kanbananza-backend/src/main/middleware/passport.js [hunk 2]
```diff
@@ -1,8 +1,6 @@
-import Users from "../models/user";
-
-const mongoose = require("mongoose");
-const passport = require("passport");
-const LocalStrategy = require("passport-local");
+import passport from "passport";
+import LocalStrategy from "passport-local";
+import User from "../models/user";
```
Comment (loucadufault) [inline kanbananza-backend/src/main/middleware/passport.js:3]: es6 imports

File: kanbananza-backend/src/main/models/board.js [hunk 3]
```diff
@@ -4,7 +4,15 @@ const Schema = mongoose.Schema;
 
 const boardSchema = new Schema({
   name: { type: String, required: true, maxLength: 100 },
-  owner: { type: Schema.ObjectId, required: true, ref: "User" },
+  ownerId: { type: Schema.ObjectId, required: true, ref: "User" },
```
Comment (loucadufault) [inline kanbananza-backend/src/main/models/board.js:7]: CHANGED (consistent with other model reference names)

File: kanbananza-backend/src/main/routes/user.js [hunk 4]
```diff
@@ -5,7 +5,7 @@ import { validateSchema } from "../middleware/schema_validator";
 
 export default (router) => {
   router.post("/user", validateSchema("createUser"), userController.create);
-  router.get("/user", userController.findByEmail);
+  router.get("/user/email/:email", userController.indexOnEmail); // see https://stackoverflow.com/a/20386425
```
Comment (RemiCarriere) [inline kanbananza-backend/src/main/routes/user.js:8]: The old acceptance tests are dependent on the previous version of this endpoint

**Review by RemiCarriere — CHANGES_REQUESTED** [review]
Looks good to me!
Once the old acceptance tests are updated to use the new user/email endpoint, please merge!

**Pull request merged**